src/valkey_search.cc

```diff
@@ -247,6 +247,7 @@ void ValkeySearch::Info(RedisModuleInfoCtx *ctx, bool for_crash_report) const {
       RedisModule_InfoAddFieldLongLong(ctx, skipped_count_str.c_str(),
                                        stat.skipped_cnt);
     };
+#ifdef VERBOSE_INFO
```

Comment:
I like Allen's suggestion for: DEBUG_INFO

```suggestion
#ifdef DEBUG_INFO
```